core/domain/expression_parser_test.py

```diff
@@ -397,18 +503,14 @@ def test_validates_math_expression(self):
 
 
         self.assertFalse(expression_parser.is_valid_expression('a+b/'))
-        self.assertFalse(expression_parser.is_valid_expression('2x + 3/2'))
+        self.assertFalse(expression_parser.is_valid_expression('a^2.'))
         self.assertFalse(expression_parser.is_valid_expression('(352+)-3*x'))
-        self.assertFalse(expression_parser.is_valid_expression('ab/2'))
-        self.assertFalse(expression_parser.is_valid_expression('Alpha/2'))
```

Comment:
Keep this; ditto below (see comment above).

Reply:
Done.